questionone

```diff
@@ -0,0 +1,21 @@
+
+function myFunction(s) {
```

Comment:
@JainVayam 

Always follow the below code standard - 
- Variable name should be specific.
- do not use any unnecessary spaces or lines.
- Always, user space after for (), if ().
- Always, space after the curly braces,s o now correct syntax would be if () {} , for () {} etc.
- Use 2 space indentation.
- Always delete the unnecessary things like console and unwanted code before pushing the code into branch.